InterventionSkillExample/main.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+from fastapi import FastAPI
+
+app = FastAPI()
+
+model = None
+
+@app.post('/invoke')
+def run(params: dict):
+
+    try:
+        payload = params["payload"]
+        profile = payload["profiles"]
+        intervention = payload["intervention_id"]
```

Comment:
Should we handle this is in a try block to account for missing parameters ? We should also, use logging as opposed to print() functions.  Use INFO level for typical log messages and WARNING/ERROR for exceptions.

Reply:
https://docs.google.com/document/d/1sP5XmxaT2DAP668XPOi9Xwjdpj6tnbGkQsR5TSGUYMQ/edit?usp=sharing Pl look at some best practices that we have captured here.  Mainly around error handling.